tests/test_websupport.py

```diff
@@ -125,56 +123,6 @@ def test_comments(support):
 
 @skip_if(sqlalchemy_missing, 'needs sqlalchemy')
 @with_support()
-def test_voting(support):
```

Comment:
re-ordering the test execution as `nose` did.
I'll rewrite such 'ordering depended test cases' at next REFACTORING PR.